bitnami/grafana/README.md

```diff
@@ -0,0 +1,180 @@
+
+## TL;DR;
+
+```console
+$ helm repo add bitnami https://charts.bitnami.com/bitnami
+$ helm install bitnami/grafana
+```
+
+## Introduction
+
+This chart bootstraps a [grafana](https://github.com/bitnami/bitnami-docker-grafana) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+Bitnami charts can be used with [Kubeapps](https://kubeapps.com/) for deployment and management of Helm Charts in clusters.
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```console
+$ helm repo add bitnami https://charts.bitnami.com/bitnami
+$ helm install --name my-release bitnami/grafana
+```
+
+These commands deploy grafana on the Kubernetes cluster in the default configuration. The [configuration](#configuration) section lists the parameters that can be configured during installation.
+
+> **Tip**: List all releases using `helm list`
+
+## Uninstalling the Chart
+
+To uninstall/delete the `my-release` statefulset:
+
+```console
+$ helm delete my-release
+```
+
+The command removes all the Kubernetes components associated with the chart and deletes the release. Use the option `--purge` to delete all persistent volumes too.
+
+## Configuration
+
+The following tables lists the configurable parameters of the grafana chart and their default values.
+
+| Parameter                                   | Description                                                                                 | Default                                                 |
+| ------------------------------------------- | ------------------------------------------------------------------------------------------- | ------------------------------------------------------- |
+| `global.imageRegistry`                      | Global Docker image registry                                                                | `nil`                                                   |
+| `global.imagePullSecrets`                   | Global Docker registry secret names as an array                                             | `[]` (does not add image pull secrets to deployed pods) |
+| `image.registry`                            | Grafana image registry                                                                      | `docker.io`                                             |
+| `image.repository`                          | Grafana Image name                                                                          | `bitnami/grafana`                                       |
+| `image.tag`                                 | Grafana Image tag                                                                           | `{TAG_NAME}`                                            |
+| `image.pullPolicy`                          | Grafana image pull policy                                                                   | `IfNotPresent`                                          |
```

Comment:
Be consistent in the descriptions. You used `Grafana image` for **image.registry** and **image.pullPolicy**, but you used `Grafana Image` for **image.repository** and **image.tag**.